### PR TITLE
Nuke: Adding `still` image family workflow

### PR DESCRIPTION
**Pull request by jakubjezek001**
Adding Still image nuke write rendering workflow from creator via publisher to loader.

# Testing notes
1. start Openpype with settings and make sure you are not having any project overides on Anatomy imageio and nuke loadImage
2. select any node in your testing workfile and go to OpenPype Menu / Create / `Create Write Still Image`. Notice the name of subset will be created regarding actual frame on timeline
3. Publish first with local setting in render knob and then with already rendered frame. 
4. Test Validator for rendered frames if you set already rendered frame and it was not rendered yet, it should be alarming your
5. Check published subset in loader and load it into scene. 
6. in Manager set to lower version then it is actually loaded into your workfile. 
7. all should be working as expected ;)

### CODE REVIEW

**Comment by jakubjezek001** [comment]
Task linked: [PYPE-1844 nuke still image family](https://app.clickup.com/t/6658547/PYPE-1844)

**Comment by mkolar** [comment]
Task linked: [PYPE-1844 nuke still image family](https://app.clickup.com/t/6658547/PYPE-1844)

**Review by mkolar — CHANGES_REQUESTED** [review]
Also I'm getting this when I try to create the still 
image

**Comment by jakubjezek001** [comment]
@mkolar did you follow the testing steps? Settings need to me treated as it is in Anatomy. Look if you see there the still write node definition in Nuke nodes ImageIO.

**Comment by mkolar** [comment]
> @mkolar did you follow the testing steps? Settings need to me treated as it is in Anatomy. Look if you see there the still write node definition in Nuke nodes ImageIO.

Thanks that was exactly it. I wiped my studio default, but project stayed. With that it works exactly as intended. I just think that the ultimate publish shouldn't have the new family, but stick to image, so we don't fragment it as much.

**Review by jrsndl — APPROVED** [review]
All works for me!

File: openpype/hosts/nuke/plugins/create/create_write_still.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+import nuke
+
+
+class CreateWriteStill(plugin.PypeCreator):
+    # change this to template preset
+    name = "WriteStillFrame"
+    label = "Create Write Still Image"
+    hosts = ["nuke"]
+    n_class = "Write"
+    family = "still"
```
[the remark this answers is not in the document]
Comment (jakubjezek001) [reply openpype/hosts/nuke/plugins/create/create_write_still.py:14]: addressed in [`8c53b3f` (#2064)](https://github.com/pypeclub/OpenPype/pull/2064/commits/8c53b3f1ce0129ee91f6d2430138cbb8e91f4c0c)

**Comment by 64qam** [comment]
The same error as mentioned above

image

**Comment by jakubjezek001** [comment]
> The same error as mentioned above
> 
> image

did you try this? https://github.com/pypeclub/OpenPype/pull/2064#issuecomment-930347375

**Review by 64qam — APPROVED** [review]
Looks good

**Review by mkolar — APPROVED** [review]

**Pull request merged**